ckanext/datatablesview/helpers.py

```diff
@@ -0,0 +1,14 @@
+from urllib.parse import quote, unquote
+from cgi import escape
+
+from ckan.plugins.toolkit import request
+
+def encode_datatables_request_filters():
+    if 'filters' not in request.params:
+        return ''
+    return escape(quote(str(request.params['filters'].encode('utf-8')).encode('ascii')))
```

Comment:
Use `request.args` instead, as that's the Flask supported name. Same in the function below